test/initializers/kafka/base-kafka-handler.test.ts

```diff
@@ -29,12 +39,12 @@ describe('base kafka handler class', async () => {
     sandbox.stub(Kafka.prototype, 'createProducer').returns(producerStub);
     sandbox.stub(Kafka.prototype, 'createConsumer').returns(consumeStub);
     const kafka = new Kafka({ groupId: 'groupId', clientId: 'clientId', brokers: [] } as any);
-    const handler = new TestKafkaHandler(kafka, { topic: 'topic', logger: getLogger('test'), batchSize: 2 });
+    const handler = new TestKafkaHandler(kafka, { topic: 'topic', logger, batchSize: 2 });
     await new Promise(resolve => setTimeout(resolve, 10));
     consumeStub.connect.calledOnce.should.eql(true);
     consumeStub.consume.calledOnce.should.eql(true);
     consumeStub.commit.calledOnce.should.eql(true);
-    handleStub.calledOnce.should.eql(true);
+    handleStub.calledTwice.should.eql(true);
```

Comment:
Why did this change?

Reply:
Maybe this should be written a little bit better. `handleStub` is called once for `handle` and once for `handleSuccess` (just before commiting we call a callback which in our case sends the output to another topic)

Reply:
What's the use of handleSuccess?